app.py

```
@@ -2078,6 +2180,7 @@ async def api_weekly_menu(req: WHOTargetsRequest) -> WeeklyMenuResponse:
 
     Returns keys: week_summary, daily_menus, weekly_coverage, shopping_list.
     """
+    global NEW_MODULAR_SYSTEM_AVAILABLE, build_week, FoodDB, RecipeDB, build_nutrition_targets
```

Flake8 (7.3.0)

[error] `global NEW_MODULAR_SYSTEM_AVAILABLE` is unused: name is never assigned in scope

(F824)